packages/@cdktf/hcl2cdk/test/granular-imports.test.ts

```diff
@@ -69,7 +69,8 @@ describe("granular-imports", () => {
     }
 `,
     [binding.kubernetes],
-    Synth.yes_all_languages,
+    Snapshot.yes,
+    Synth.yes_but_only_typescript_right_now_because_it_breaks,
```

Comment:
Because it never worked to begin with, it's just not complete. A while back when doing 0.16 I started working on getting synth working for JSII languages and then priorities changed and I had to abandon the PR prior to completion. We got it in anyways because we needed some adjacent refactorings and the work for synth was okay, just not complete. It was skipped at that time, the PR makes it just more obvious that it's skipped right now.